samples/deployment/test-network/README.md

```diff
@@ -26,68 +27,52 @@ Use `CC_ID` and `CC_PATH` to define the FPC Chaincode you want to build.
 cd $FPC_PATH/samples/deployment/test-network
 export CC_ID=echo
 export CC_PATH=$FPC_PATH/samples/chaincode/echo
+# TODO this needs to be changed!
+export CC_VER=$(cat CC_PATH/)
```

Comment:
This does not look right -- cat of a path?!

Reply:
Yes, that is why there is a TODO on top ....

Reply:
resolved